specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-05-01-preview/containerregistry_scopemap.json

```diff
@@ -886,26 +913,58 @@
         }
       }
     },
-    "ProxyResource": {
-      "description": "The resource model definition for a ARM proxy resource. It will have everything other than required location and tags.",
+    "SystemData": {
```

Comment:
Reason for adding local SystemData model:
OAV tool has a bug which causes model validation failure upon referencing SystemData from ARM common-types.
Therefore, ARM team has suggested using local models to circumvent this issue.
Bug: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1085423, 
https://github.com/Azure/oav/issues/280